src/Dependencies/Collections/Internal/RoslynUnsafe.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.CodeAnalysis.Collections.Internal
+{
+    internal static unsafe class RoslynUnsafe
+    {
+        /// <summary>
+        /// Returns a by-ref to type <typeparamref name="T"/> that is a null reference.
+        /// </summary>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static ref T NullRef<T>()
+        {
+            return ref Unsafe.AsRef<T>(null);
+
+            // ldc.i4.0
```

Comment:
Do we need or want to keep these il comments?

Reply:
I'll prune/clean this file since it's unlikely we'll need to synchronize it in the future.